test/expected.txt

```diff
@@ -290,6 +293,54 @@
 || 
 || 2 persons listed!
 || ===================================================
+|| Enter command: || [Command entered:  edit]
+|| Invalid command format!
```

Comment:
Good job for handling i/o tests. Next, try to familiarize yourself with JUnit tests!